### ISSUE TITLE
TypeError: Cannot read property 'split' of null

### ISSUE THREAD

**Issue by franzwarning** [invertase/react-native-firebase#2766]
I found a bug when trying to `firebase.auth().verifyPhoneNumber` on iOS & Android with a bad phone number. If the verification fails, the `_verificationFailedHandler` in `auth/lib/PhoneAuthListener.js` creates a new `NativeFirebaseError` like so:
```
snapshot.error = new NativeFirebaseError({ userInfo: state.error }, null, 'auth');
```
The issue with this is the null parameter passed into the `jsStack` value in the NativeFirebaseError constructor. Later on in the NativeFirebaseError.js file we try to call
```
return [message, ...this.jsStack.split('\n').slice(2, 13)].join('\n');
```
... and because this.jsStack is null the error arises. Any ideas on the best way to fix this?

**Comment by Ehesp** [comment]
Could you should us the JS code you're using? Also what RNFB version?

**Comment by franzwarning** [comment]
Fixed in 6.0.2
